src/App.jsx

```diff
@@ -1,34 +1,32 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
-
-function App() {
-  const [count, setCount] = useState(0)
+import { Notification } from "./components/notification";
+import styles from './App.module.css';
+import { IconCheck } from "./components/notification/icons/IconCheck";
+import { IconInfo } from "./components/notification/icons/IconInfo";
+import { IconError } from "./components/notification/icons/IconError";
 
+export function App() {
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
+    <>
+      <div className={styles.notification}>
+        <div className={styles.title}>Notifications</div>
+        <Notification
+          text='Notification'
+          color={styles.warning}
+          icon={<IconInfo />}
+        />
+        <Notification
+          text='Notification'
+          color={styles.success}
+          icon={<IconCheck />}
+        />
+        <Notification
+          text='Notification'
+          color={styles.danger}
+          icon={<IconError />}
+        />
```

Comment:
```suggestion
        <Notification type="warning">Notification</Notifications>
        <Notification type="success">Notification</Notifications>
        <Notification type="error">Notification</Notifications>
```

Треба зробити так, щоб потім полегшити собі життя! Навіщо Icon і color, якщо можна передати type і вже в самому компоненті міняти кольори і іконки!
також намагайся контент вставляти через children